.github/workflows/test.yml

```diff
@@ -114,4 +114,5 @@ jobs:
       if: steps.check-key.outputs.api_key_available == 'false'
       run: |
         echo "🔒 Simulation tests skipped (no API keys configured)"
-        echo "To enable simulation tests, add GEMINI_API_KEY and/or OPENAI_API_KEY as repository secrets"
\ No newline at end of file
+        echo "To enable simulation tests, add GEMINI_API_KEY and/or OPENAI_API_KEY as repository secrets"
```

Comment:
Live tests were removed and replaced with actual simulation tests that can be run directly by the user, please review and remove this